src/scalar.rs

```diff
@@ -568,6 +571,36 @@ impl Zeroize for Scalar {
     }
 }
 
+#[cfg(all(feature = "ff", feature = "rand_core"))]
```

Comment:
Would be nice to use namespaced features so an `ff` feature could activate `dep:ff` and `rand_core`

Reply:
We will be using this in `ff` itself once its MSRV is 1.60 (https://github.com/zkcrypto/ff/issues/78), but I haven't seen a strong push for that MSRV yet in the wider ecosystem.